src/ts/metatxhandler.ts

```diff
@@ -49,379 +29,140 @@ export enum ChainID {
  * replay protection.
  */
 export class MetaTxHandler {
-  public static getHubAddress(chainid: ChainID, contractType: ContractType) {
+  public static getForwarderAddress(
```

Comment:
This function is actually incorrectly named, but I guess it will dissappear when you add the ForwarderFactories.

Reply:
its gone now

Reply:
oh wait, you mean getForwarderAddress()? 

I made this a protected method. Since it is used internally by the factory.

Reply:
Yeah, i think it's incorrectly names as for the relay hub it does get the forwarder address, but for the proxy account factories it gets the factory address, not the forwarder address. I mentioned in slack that this should be split up into the forwarder factory classes, so that they dont have to look at each others addresses.

Reply:
the getAddresses() have been tided up since this comment. 

getAddress() for the proxyForwarder returns the ProxyAccount
getAddress() for the relayhub returns the RelayHub